### DISCUSSION TITLE
color bar not shown in VMware

### DISCUSSION THREAD

**Discussion by mellow753** [pyvista/pyvista#2307] [Q&A]
Hi, I use pyvista in VM Ubuntu, and it seems that the color bar cannot be rendered normally, and I have the figure below:
![U_0 6](https://user-images.githubusercontent.com/57876661/157152913-e3529f1b-127f-4968-af05-7d9c2b2c0663.png)
Can anyone tell me how to solve this problem?

**Comment by banesullivan** [comment]
This is known and encountered in https://github.com/pyvista/pyvista/issues/48

**Reply by mellow753** [reply]
Thanks for reply! So I guess there is still no solution for this problem up to now.

**Reply by banesullivan** [reply]
Not that I am aware of at least... sorry I couldn't help with this. It's an annoying issue

**Reply by mellow753** [reply]
Thanks for your clarification!
